cachi2/core/package_managers/bundler/main.py

```diff
@@ -99,3 +106,36 @@ def _get_repo_name_from_origin_remote(package_dir: RootedPath) -> str:
 
     resolved_path = Path(repo_name).joinpath(package_dir.subpath_from_root)
     return str(resolved_path)
+
+
+def _prepare_environment_variables_for_hermetic_build() -> list[EnvironmentVariable]:
+    return [
+        # Contains path to a directory where a new config could be found.
+        EnvironmentVariable(name="BUNDLE_APP_CONFIG", value="${output_dir}/" + CONFIG_OVERRIDE),
+    ]
+
+
+def _prepare_for_hermetic_build(source_dir: RootedPath, output_dir: RootedPath) -> None:
+    """Prepare a package for hermetic build by injecting necessary config."""
+    potential_bundle_config = source_dir.join_within_root(".bundle/config").path
+    hermetic_config = dedent(
+        f"""
+        BUNDLE_CACHE_PATH: "{output_dir}/deps/rubygems"
```

Comment:
You'll need `${output_dir}` here as well.

Reply:
Done, thank you.